ReactiveFeedback/Feedback.swift

```diff
@@ -5,6 +5,29 @@ import enum Result.NoError
 public struct Feedback<State, Event> {
     public let events: (Scheduler, Signal<State, NoError>) -> Signal<Event, NoError>
 
+
+    /*
```

Comment:
The docs for the feedback initialisers do not show up.

Reply:
Yeah seems that Xcode can node parse `/* */` comments for initializers.
Used `///` instead